.github/workflows/ci-pre-commit.yml

```diff
@@ -0,0 +1,16 @@
+name: linting
+
+on:
+  push:
+    branches: "*"
+  pull_request:
+    branches: "*"
+
+jobs:
+  linting:
+    name: "pre-commit hooks"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - uses: pre-commit/action@v2.0.0
```

Comment:
Note that this enforces all code formatters we have in our `pre-commit` configuration, which means we now also enforce `blackdoc`.

Reply:
FWIW, I would support this change. Its very nice bit of consistency to pair pre-commit linting with enforcement.